web/web.go

```diff
@@ -297,6 +300,11 @@ func New(logger log.Logger, o *Options) *Handler {
 			return *h.config
 		},
 		o.Flags,
+		api_v1.Options{
```

Comment:
`Options` is a bit generic of a name in this context, given that all the other parameters to `NewAPI()` are also options of sorts. Maybe eventually it'd be good to put everything into one big options struct, like we do in `web.go` for the web handler. But how about for now, calling it `GlobalURLOptions` or something like that?

Reply:
Sure. Done.